cmake/caches/Windows-aarch64.cmake

```diff
@@ -26,15 +26,27 @@ set(LLVM_DEFAULT_TARGET_TRIPLE aarch64-unknown-windows-msvc CACHE STRING "")
 set(LLVM_APPEND_VC_REV NO CACHE BOOL "")
 set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR YES CACHE BOOL "")
 set(LLVM_ENABLE_PYTHON YES CACHE BOOL "")
-set(LLVM_RUNTIME_TARGETS
-      aarch64-unknown-windows-msvc
-    CACHE STRING "")
+
+set(DEFAULT_RUNTIME_TARGETS
+      aarch64-unknown-windows-msvc)
```

Comment:
Hmm, I wonder why we don't have `x86_64-unknown-windows-msvc` listed here. I suppose that we can fix that in a follow up.

Reply:
I may have heard that that's just because it has been added yet?